src/signals/incident/components/form/MapSelectors/components/MapMessage/MapMessage.tsx

```diff
@@ -100,6 +100,12 @@ export const MapMessage: FC<MapMessageProps> = ({
 }) => (
   <MessageOverlay {...props} data-testid="mapMessage" type="map">
     {children}
-    <CloseButton onClick={onClick} icon={<Close />} variant="blank" aria-label="Melding sluiten" />
+    <CloseButton
+      data-testid="closeMessage"
```

Comment:
in this project we should use either kebab case or camelcase. Maybe choose one instead. I putted a point on the list here: I would suggest kebab, just like in most cypress tests and because is doesnt look like a js variable then.
https://datapunt.atlassian.net/browse/SIG-4802?atlOrigin=eyJpIjoiN2ZjNzMzOGM2YTAxNDEzNWI4MTE3MWNlOTMwNDAxYmEiLCJwIjoiaiJ9